integration_test.go

```diff
@@ -137,3 +137,12 @@ func TestTestsTs(t *testing.T) {
 		t.Fatal(err.Error())
 	}
 }
+
+func TestErrorFilename(t *testing.T) {
+	integrationTestSetup()
+	cmd := exec.Command(denoFn, "*errorName.ts")
```

Comment:
Using * isn't great because it the meaning of wildcard - even though it won't be interpreted as such here. Suggest `nonexistentScript.ts`